crates/solidity/outputs/cargo/slang_solidity/Cargo.toml

```diff
@@ -33,7 +33,9 @@ required-features = ["cli"]
 
 [features]
 default = ["cli"]
-cli = ["dep:anyhow", "dep:clap", "dep:serde_json"]
+cli = ["dep:anyhow", "dep:clap", "dep:serde_json", "__private_ariadne"]
+# This is meant to be used by the CLI or internally only.
```

Comment:
question: I wonder why are we adding an extra feature here, instead of just adding `dep:ariadne` to the existing default `cli` feature.

Reply:
Per the comment, because it's also used internally (by our other crates crates in the workspace) and not only via the CLI, see https://github.com/NomicFoundation/slang/pull/963/files#diff-059ed3ad7d6d34ada23bb6a205ff05636f0f434ff9fa8ebf7c1a0c596b81b893R22 for instance.